app/services/idv_attempter.rb

```diff
@@ -0,0 +1,34 @@
+class IdvAttempter
+  attr_accessor :current_user, :user_session
```

Comment:
Both accessors are used externally in the specs to set context. See e.g. 47c50884faebbab29916068b513f2d38c218f31d